lib/ledger.js

```diff
@@ -146,9 +145,12 @@ async function _setupGenesisNode() {
     });
     const ledgerOwner = await _getLedgerOwner();
     // generate the Maintainers's DID Document
-
-    const maintainerDoc = await didv1.generate(
-      {passphrase: config['veres-one'].maintainerPassphrase});
+    // this needs to be an Ed22519VerificationKey2020
+    const keyOptions = {
+      seed: Buffer.from(config['veres-one'].maintainerPassphrase)
```

Comment:
This needs at least a FIXME. We shouldn't generate a seed from a short maintainer passphrase. We either need a very long passphrase that we hash appropriately or we need to use a salt+b/scrypt -- if we're going to produce a seed from human readable text.

Reply:
changed it to use a `sha-256` hash: https://github.com/veres-one/veres-one/pull/44/commits/01b3c12a00d416d83f10d62240bf54000a07722a